infrastructure/kube/keep-test/maintainer/bitcoin-difficulty-maintainer-service.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Service
```

Comment:
Do we need to expose the maintainer as a service?

Reply:
I don't think so.
Removed the whole file in f8516893b7f34638e1465e5c7116cd6d8a10aed8.